src/fckit/module/fckit_configuration.cc

```diff
@@ -342,7 +342,7 @@ int32 c_fckit_configuration_get_array_string( const Configuration* This, const c
         offsets[j] = size;
         size += s[j].size();
     }
-    value = new char[size];
+    value = new char[size + 1];
```

Comment:
Here I can understand, I think.
strcpy always adds the extra "null-termination" character.
For each subsequent `j`-iteration this character gets overwritten, except for the last iteration. Hence the +1 required in allocation.

Reply:
Good, we can merge this one.